docs/docs/atom-archive/getting-started/sections/installing-atom.md

```diff
@@ -0,0 +1,264 @@
+$ sudo sh -c 'echo -e "[Atom]\nname=Atom Editor\nbaseurl=https://packagecloud.io/AtomEditor/atom/el/7/\$basearch\nenabled=1\ntype=rpm-md\ngpgcheck=0\nrepo_gpgcheck=1\ngpgkey=https://packagecloud.io/AtomEditor/atom/gpgkey" > /etc/zypp/repos.d/atom.repo'
+$ sudo zypper --gpg-auto-import-keys refresh
+```
+
+You can now install Atom using `zypper`:
+
+``` command-line
+# Install Atom
+$ sudo zypper install atom
+
+# Install Atom Beta
+$ sudo zypper install atom-beta
+```
+
+Alternatively, you can download the [Atom .rpm package](https://atom.io/download/rpm) and install it directly:
+
+``` command-line
+$ sudo zypper in -y atom.x86_64.rpm
+```
+
+{{/linux}}
+
+#### Updating Atom
+
+You should consider updating Atom periodically for the latest improvements to the software.  Additionally, When Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible.
+
+{{#mac}}
+
+"Automatically Update" is enabled by default in Core Settings of the [Settings View](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences), which will allow Atom to check for updates automatically. If you disable this setting you can update Atom manually.
+
+To perform a manual update:
+
+* Click on the `Atom > Check for Update` menu item in the menu bar.
+* Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button.
+
+Atom will begin to update if an update is available.
+
+{{/mac}}
+
+{{#windows}}
+
+"Automatically Update" is enabled by default in Core Settings of the [Settings View](https://flight-manual.atom.io/getting-started/sections/atom-basics/#settings-and-preferences), which will allow Atom to check for updates automatically. If you disable this setting you can update Atom manually.
+
+To perform a manual update:
+
+* Click on the `Help > Check for Update` menu item in the menu bar.
+* Search for `Application: About` in the [Command Palette](https://flight-manual.atom.io/getting-started/sections/atom-basics/#command-palette) and click the `Check now` button.
+
+Atom will begin to update if an update is available.
+
+{{/windows}}
+
+{{#linux}}
+
+If you are using Atom's official package repositories, use your distribution's package manager to update Atom.  Otherwise, you will need to manually download and install the latest `.rpm` or `.deb` package from https://atom.io. For more details, see [Installing Atom on Linux.](https://flight-manual.atom.io/getting-started/sections/installing-atom/#installing-atom-on-linux)
+
+{{/linux}}
+
+#### Portable Mode
+
+Atom stores configuration and state in a `.atom` directory usually located in your home directory <span class="platform-windows">(`%userprofile%` on Windows)</span>. You can however run Atom in portable mode where both the app and the configuration are stored together such as on a removable storage device.
```

Comment:
Again, I find this a tricky one. Sure, HTML is valid inside Markdown, but I still think it should be avoided whenever possible. Especially when all it does is add a class, which has no purpose in a pure Markdown.

Reply:
I think this is just an artefact that needs to be removed in general. It was used for the "dynamic" flight manual where that class would only be displayed if the selected platform = windows.